testutils/pytest.py

```diff
@@ -107,3 +107,13 @@ def check_rc(only_rc_allowed):
             reason="RIOT version under test is not a release candidate"
         )
     return rc_only_mark
+
+
+def get_required_envvar(envvar):
+    """
+    Returns the value of an environment variable. Raise RuntimeError otherwise.
+    """
+    res = os.environ.get(envvar)
+    if not res:
+        raise RuntimeError("Missing {} in env variables".format(envvar))
```

Comment:
Why is a `RuntimeError` required in case an environment variable required? I believe the usual `KeyError` is sufficient, making this function unnecessary. E.g.

```
@pytest.fixture
def nwkskey():
    return os.environ["LORAWAN_NWK_SKEY"]
```

Reply:
I did it just to show the error message. With this it just shows "KeyError: LORAWAN_NWK_SKEY", but it's not that obvious that this is an env variable (for someone that manually runs tox).
But we could replace that with good documentation

Reply:
In that case I would prefer to make this function more pythonic:

```py
    try:
        return os.environ.get(envvar)
    raise KeyError:
        raise RuntimeError("Missing {} in env variables".format(envvar))
```

Reply:
os.environ.get doesn't return error :(
```
Python 3.9.0 (default, Oct  7 2020, 23:09:01) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ.get("LALALA")
>>> 
```

Reply:
what should I do with this one then?

Reply:
Arghs sorry, I misstyped:

```py
    try:
        return os.environ[envvar]
    raise KeyError:
        raise RuntimeError("Missing {} in env variables".format(envvar))
```

Reply:
done!